### PR TITLE
[libraries/BVL_Feedback_Panel] $sessionID must be type SessionID

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

A string was given and this change resolves it.

### CODE REVIEW

**Comment by laemtl** [comment]
There is another [getFeedbackPanel here](https://github.com/aces/Loris/blob/5166ada28fbb01ac020f03c6eb38766f2ff7e32a/php/libraries/NDB_BVL_Instrument.class.inc#L2415). I wonder if the 2 signature should agree, it will be confusing otherwise.

If so, the sessionID type change should be done before calling getFeedbackPanel.

**Comment by laemtl** [comment]
Related to #7063

**Comment by laemtl** [comment]
Just noticed the [signature](https://github.com/aces/Loris/blob/4af1256b6072ec79e08870dc1f44e851b343600d/modules/instrument_list/php/instrument_list.class.inc#L293), sessionID should be of type SessionID, so this function is probably called somewhere with the wrong argument. Maybe also specifying the types in the function definition will help to clarify the confusion.

**Comment by driusan** [comment]
@maltheism Can you rebase so that this runs with php8 and to ensure all the latest phan fixes still pass?

**Review by kongtiaowang — APPROVED** [review]
LGTM

File: modules/instrument_list/php/instrument_list.class.inc [hunk 1]
```diff
@@ -296,7 +296,10 @@ class Instrument_List extends \NDB_Menu_Filter
      */
     function getFeedbackPanel($candID, $sessionID): string
```
Comment (laemtl) [inline modules/instrument_list/php/instrument_list.class.inc:297]: The signature (sessionID) disagrees with the current change.
This function is called somewhere with the wrong argument. Specifying the types in the function definition will help to clarify the confusion.

**Review by laemtl — CHANGES_REQUESTED** [review]

**Comment by laemtl** [comment]
@maltheism I submitted a commit with suggested changes. Let me know what you think.
Also, I investigated I couldn't find an call of getFeedbackPanel with $sessionID of type string.

**Comment by driusan** [comment]
Our tests aren't amazing at finding mismatches because `PhanTypeMismatchArgument` is still ignored. After #7322 I have another PR in the pipeline that fixes the remaining `PhanTypeMismatchArgument` errors and unignores it.

In the meantime, maybe adding `declare(strict_types=1);` to the file with the signature change would help the tests find any culprits calling it with a string?

**Comment by laemtl** [comment]
@driusan I added declare(strict_types=1) and it confirms my suspicion: The code breaks when we call 
`new \SessionID($sessionID)` with ` PHP Fatal error:  Uncaught TypeError: Argument 1 passed to ValidatableIdentifier::__construct() must be of the type string, object given`.

**Comment by laemtl** [comment]
Since the strict _types flag, Phan is complaining with 
` modules/instrument_list/php/instrument_list.class.inc:255 PhanTypeMismatchArgumentProbablyReal Argument 1 ($select_inactive_threads) is $feedback_select_inactive of type ?'Y' but \NDB_BVL_Feedback::getMaxThreadStatus() takes bool (no real type) defined at php/libraries/NDB_BVL_Feedback.class.inc:380 (the inferred real argument type has nothing in common with the parameter's phpdoc type)`

**Comment by driusan** [comment]
That sounds a lot like one of the errors I had to fix while fixing the phan errors.

**Comment by laemtl** [comment]
@driusan I tried something and it fixes phan.

**Comment by driusan** [comment]
The 'Y' => true thing looks familiar. I think that's what I had to do to fix it too..

**Review by laemtl — APPROVED** [review]

**Pull request merged**